Libraries/plist/CMakeLists.txt

```diff
@@ -69,6 +69,11 @@ add_executable(plutil Tools/plutil.cpp)
 target_link_libraries(plutil plist process util)
 install(TARGETS plutil DESTINATION usr/bin)
 
+add_executable(PlistBuddy Tools/PlistBuddy.cpp ${CMAKE_SOURCE_DIR}/ThirdParty/linenoise/linenoise.c)
+target_include_directories(PlistBuddy PRIVATE "${CMAKE_SOURCE_DIR}/ThirdParty/linenoise")
+target_link_libraries(PlistBuddy plist util)
+install(TARGETS PlistBuddy DESTINATION usr/bin)
```

Comment:
Made optional in 96ed319dcc01244e73dd4dd872b9392c1df1c6ef. Falls back to `fgets()` when not available.